background_b: 255
background_g: 86
background_r: 250
rosdistro: 'kinetic

  '
roslaunch:
  uris: {host_localhost__36165: 'http://localhost:36165/'}
rosversion: '1.12.17

  '
run_id: 52f96486-59ae-11ec-addc-000c29eacc5f
